CppSamples/CloudAndPortal/SearchForWebmap/SearchForWebmap.qml

```diff
@@ -82,6 +81,7 @@ SearchForWebmapSample {
     BusyIndicator {
         anchors.centerIn: parent
         running: !portalLoaded
+        visible: !portalLoaded
```

Comment:
Are you certain this is needed? This PR has a few of these.

https://doc.qt.io/qt-6.8/qml-qtquick-controls-busyindicator.html#:~:text=Note%3A%20The%20indicator%20is%20only%20visible%20when%20this%20property%20is%20set%20to%20true.